job/job_test.go

```diff
@@ -34,13 +35,15 @@ func getMockJobWithGenericSchedule() *Job {
 }
 
 func TestScheduleParsing(t *testing.T) {
+	cache := NewMockCache()
+
 	fiveMinutesFromNow := time.Now().Add(
 		time.Duration(time.Minute * 5),
```

Comment:
@ajvb 5 minutes already is a duration; `time.Now().Add(5*time.Minute)` will do just fine :-)